### PR TITLE
Feature-child-gameplay-styles

### PR DESCRIPTION
**Pull request by Julianisrel**
{Child Path Footer: Remove links/recenter copyright}
   * Remove links from the footer
   * Right align the copyright to replace where the links were

### CODE REVIEW

**Review by mollybee — DISMISSED** [review]
Great! You got rid of that little random 'main' floating around there! Looks good.

File: src/styles/less/Hud.less [hunk 1]
```diff
@@ -17,6 +18,11 @@
     flex-direction: column;
 }
 
+<<<<<<< HEAD
```
Comment (Fry-Fr) [inline src/styles/less/Hud.less]: Are these lines conflicts that still need to be resolved?

File: src/styles/less/Hud.less [hunk 2]
```diff
@@ -31,7 +37,11 @@
 .ctimer.pos-2 {
     margin-left: 33.33%;
     width: 50%;
+<<<<<<< HEAD
```
Comment (Fry-Fr) [inline src/styles/less/Hud.less]: looks like more conflicts with merging. lines should be cleared.

File: src/styles/less/Hud.less (hunk 1 above)
Comment (rhiannonstanford) [reply src/styles/less/Hud.less]: resolved

File: src/styles/less/Hud.less (hunk 2 above)
Comment (rhiannonstanford) [reply src/styles/less/Hud.less]: thank you for pointing this out, resolved.

**Review by rhiannonstanford — APPROVED** [review]
Files approved.  Merge conflict lines deleted.

File: src/components/pages/GamePlay/Footer.js [hunk 3]
```diff
@@ -8,30 +8,7 @@ export default function Footer() {
...
-              Privacy
-            </Link>
-          </div>
-          <div className="footer-nav-div-link">
-            <Link className="footer-nav-link" to="#">
-              Terms
-            </Link>
-          </div>
-        </nav>
+        <div class="copyright">©{curYear} Story Squad HQ</div>
```
Comment (Alisa1989) [inline src/components/pages/GamePlay/Footer.js:11]: Good clean up

**Review by Alisa1989 — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Good job removing the links from the child footer!

**Pull request merged**